### ISSUE TITLE
Article scrolling causes reload button to flicker

### ISSUE THREAD

**Issue by samcat116** [Ranchero-Software/NetNewsWire#3345]
Running version 6.0.3 on macOS Monterey. Scrolling while in dark mode can cause the area around the reload button to become darker. Moving the cursor near this area fixes the issue, but it will happen again once you start scrolling. I've attached a video showing it in action.

https://user-images.githubusercontent.com/11448286/140934470-a1e40f79-63f9-42b5-8b91-357dcb1c38ae.mov

**Comment by vincode-io** [comment]
What do you have behind your window that is causing your sidebar to be that color?

**Comment by samcat116** [comment]
This was in full screen mode. My desktop wallpaper is a night photo.

**Comment by vincode-io** [comment]
Yeah, this looks like some kind of weird interaction between WKWebView, NSToolbar, and fullscreen mode. Since these are all OS provided services, there really isn't anything I can think of that we can do to mitigate this.

**Comment by brentsimmons** [comment]
Is this still a bug?

**Comment by brentsimmons** [comment]
(Closed it accidentally.)
